project/CopyrightHeader.scala

```diff
@@ -64,10 +64,36 @@ trait CopyrightHeader extends AutoPlugin {
       case _                                          =>
         header
     }
+    
+    def parseStartAndEndYear(header:String):Option[(String,Option[String])] = header match {
+      case CopyrightHeaderPattern(years, null, _)     =>
+        Some((years,None))
+      case CopyrightHeaderPattern(years, endYears, _) =>
+        Some((years,Some(endYears)))
+      case _                                          =>
+        None
+    }
 
     override def apply(text: String, existingText: Option[String]): String = {
-      val updatedExistingText = existingText.map(updateLightbendHeader)
-      HeaderCommentStyle.cStyleBlockComment.commentCreator(text,updatedExistingText)
+      val formatted = existingText match {
```

Comment:
Cause the original code generates `/** ...` for header, so I have to change the code here to update the old headers.